paasta_tools/setup_kubernetes_cr.py

```diff
@@ -175,7 +174,7 @@ def setup_all_custom_resources(
             ).items
         except ApiException:
             log.debug(
-                "Listing CRDs with apiextensions/v1 not supported on this cluster, falling back to v1beta1"
+                "Listing CRDs with apiextensions/v1 not supported on this cluster"
```

Comment:
imo: i think we'd want to revert the changes to this file where we added the for loop rather than make this a for loop over a single version

Reply:
(it would make the code more straightforward and while we may need to handle new apiextension versions in the future, imo it'd be better to only temporarily have this sort of multi-version code)